modules/rosa-installing-prerequisites.adoc

```diff
@@ -241,10 +237,21 @@ I: Verifying whether OpenShift command-line tool is available...
 E: OpenShift command-line tool is not installed.
 Run 'rosa download oc' to download the latest version, then add it to your PATH.
 ----
+
+. Install the OpenShift Container Platform command-line interface (`oc` CLI) from the `rosa` CLI.
+.. Enter this command to download the latest version of the `oc` CLI:
 +
-[NOTE]
-====
-If you have not already installed the OpenShift Command-line Interface (CLI), also known as `oc`, run `rosa download oc` to download the latest version, then add it to your PATH.
-====
+[source,terminal]
+----
+$ rosa download oc
+----
+
+.. After downloading the `oc` CLI, unzip it and add it to your path.
+.. Enter this command to verify that the `oc` CLI is installed correctly:
++
+[source,terminal]
+----
+$ rosa verify oc
+----
 
-After completing these steps you are ready to create an {product-title} cluster.
+After completing these perequisite steps, you are ready to create an {product-title} cluster.
```

Comment:
perequisite -> prerequisite

Also, can you remind me what {product-title} resolves to for ROSA? (Asking because of the "an" in front) I'm a little confused because I'm looking for the common attributes file, which I see here [1] I think, but that says OpenShift Dedicated.

[1] https://github.com/openshift/managed-openshift-docs/blob/4.5/_attributes/attributes-openshift-dedicated.adoc

Reply:
The name did change to Red Hat OpenShift Service on AWS, so now it should be "a" before {product-title}.

Reply:
And I did change it to "a".